docs/content/preview/develop/common-patterns/timeseries/ordering-by-entity.md

```diff
@@ -104,23 +106,27 @@ Because the default value of `bucketid` is set to `random()*8`, you do not have
 Retrieve the data from the table as follows:
 
 ```sql
-SELECT *, yb_hash_code(car,bucketid) % 3 as tablet FROM entity_order2;
+SELECT * FROM entity_order2;
 ```
 
 ```output
-         ts          |  car  | speed | bucketid | tablet
----------------------+-------+-------+----------+--------
- 2023-07-01 00:00:21 | car-1 |    45 |        7 |      2
- 2023-07-01 00:00:22 | car-1 |     9 |        7 |      2
- 2023-07-01 00:00:37 | car-1 |    32 |        7 |      2
- ...
- 2023-07-01 00:00:27 | car-2 |    26 |        1 |      2
- 2023-07-01 00:00:40 | car-2 |    26 |        1 |      2
- 2023-07-01 00:01:10 | car-2 |    46 |        1 |      2
- ...
- 2023-07-01 00:00:08 | car-1 |    43 |        2 |      1
- 2023-07-01 00:00:13 | car-1 |    20 |        2 |      1
- 2023-07-01 00:00:20 | car-1 |    53 |        2 |      1
+         ts          |  car  | speed | bucketid
+---------------------+-------+-------+----------
+ 2023-07-01 00:00:06 | car-1 |     4 |        7
+ 2023-07-01 00:00:09 | car-1 |    55 |        7
+...
+ 2023-07-01 00:00:53 | car-1 |     5 |        7
+ 2023-07-01 00:01:05 | car-1 |     9 |        7
+ 2023-07-01 00:00:14 | car-2 |    29 |        1
+...
+ 2023-07-01 00:01:00 | car-1 |    24 |        2
+ 2023-07-01 00:01:37 | car-1 |    13 |        2
+ 2023-07-01 00:00:11 | car-2 |    30 |        6
+ 2023-07-01 00:00:30 | car-2 |    30 |        6
+...
+ 2023-07-01 00:01:35 | car-2 |    14 |        6
+ 2023-07-01 00:00:31 | car-2 |    55 |        0
+ 2023-07-01 00:00:44 | car-2 |    45 |        0
 ```
 
 Notice that the data for each car is split into buckets and that the data is ordered on by `ts` in each bucket, and that the buckets are distributed across different tablets.
```

Comment:
With the tablet column gone, how do I know the buckets are distributed across tablets?

Reply:
I was asked to remove the tablet column as it was unnecessary. We will just mention that the buckets are distributed across tablets.